### ISSUE TITLE
Add markdown rendering to comments as is done with the description field in card visualization

### ISSUE THREAD

**Issue by victorkane** [basementuniverse/vscode-kanbn#5]
Hi there. First of all, I'm really amazed at the thought and feature set that makes up vscode-kanbn, which it achieves while at the same time maintaining a huge degree of both simplicity and stability. At the risk of repeating myself, as a decades old user of products like [Pivotal Tracker](https://www.pivotaltracker.com/) and [Trello](https://trello.com/), and more recently the open source [Kanboard](https://kanboard.org/), I am excitedly using vscode-kanbn on a daily basis in all of my projects and am thrilled at last to have my process linked to my codebase, not only in the repository, but also in the vscode editing experience.
This issue is a feature request: In the same way that there is an editing mode for descriptions as well as a markdown rendered presentation mode, it would be great if comments were also markdown based and rendered when not being editing.
This would be great for any use case. My use case is to be able to add comments used as sort of "mini-descriptions", as I complete the sub-tasks of a card.
If you take a look at pivotal tracker, what I am requesting here would just about place vscode-kanbn on an equal footing with pivotal tracker despite its simplicity, openness and lack of vendor-lock-in or dependencies.
Thanks again, for a great project!

**Comment by basementuniverse** [comment]
Hi @victorkane, thank you very much for your comments!

To some extent, this project is a proof-of-concept (I like to call it "kanban-as-code", similar to how we've already got "testing-as-code", "infrastructure-as-code", "documentation-as-code" etc.) and your feedback serves as positive validation for the concept :smile:

Regarding the feature request: I think this is a good idea and it would definitely be a nice feature to have. Rendering comments as markdown should be quite easy to add to the extension.

However, the subset of markdown that can appear inside comments might be limited because they're stored as lists, e.g.:

```markdown
## Comments

- date: 2021-05-06T09:43:33.802Z
  First comment...
- author: Test
  date: 2021-05-06T09:44:03.054Z
  Second comment...
```

If this format is too limited, maybe it would be possible to change the format (with backwards-compatibility of course) to allow more markdown. I'll do some testing and experiments, and hopefully have an update ready soon.

Thanks! :smile:

**Comment by basementuniverse** [comment]
I have released version 0.9.0 of vscode-kanbn - it now renders markdown in comments alongside a few other minor improvements.

There was an issue where headings inside comments would be incorrectly copied into the task’s description field, it was due to the way I was parsing markdown to json, but this has been fixed (and as an added bonus the kanbn project has 1 less dependency now 😄)

Tests are all passing, so it should all be working nicely, but if you notice any bugs please let me know!

Thanks again for your feedback @victorkane
